Documentation CYF 1.0/pages/api-functions-waves.html

```diff
@@ -171,6 +171,12 @@ <h3>The Arena Object</h3><br><br>
                 <li><span class="CYF"></span> <span class="term">Arena.Show()</span> - Makes the Arena visible after using
                 <span class="term">Arena.Hide</span>.<br>
                 This is also called automatically whenever a wave ends!</li><br>
+
+                <li><span class="new066"></span> <span class="CYF"></span> <span class="term">Arena.SetSprite(<span class="string">filename</span>)</span> - Set Arena's sprite.</li><br>
```

Comment:
This doesn't explain what sprite it changes; inner or outer? How does this work with scaling?